### DISCUSSION TITLE
r/city-china-zhejiang-shaoxing-pingwangxian

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#36] [Announcements]
# r/city-china-zhejiang-shaoxing-pingwangxian

游记 | 自驾冬日里的绍兴平王线

https://blog.ops-coffee.cn/r/city-china-zhejiang-shaoxing-pingwangxian

**Comment by xyzbz** [comment]
好生活哈

**Reply by ops-coffee** [reply]
😄

**Comment by winjune** [comment]
从少数派房车用电的文章过来，在绍兴十多年，还没开过平王线，准备去开一下~~

**Reply by ops-coffee** [reply]
绍兴可玩儿性很高，好多不错的线路，都可以慢慢跑跑
